### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: '/thesillyhome_src/data/parsed/act_states.pkl'

### ISSUE THREAD

**Issue by dbabin** [lcmchris/thesillyhome-container#33]
I am getting the above error. Here is a copy of my log. 

I am also getting a few Warnings. Included in log below. 

2022-09-20 11:39:55.233022 WARNING model_executor: ------------------------------------------------------------
2022-09-20 11:39:59.128560 INFO model_executor: 
2022-09-20 11:39:59.132359 INFO model_executor: <--- sensor.shellypro4pm_c8c9a3efe564_switch_1_energy is 9.06 --->
2022-09-20 11:39:59.133859 WARNING model_executor: ------------------------------------------------------------
2022-09-20 11:39:59.135004 WARNING model_executor: Unexpected error in worker for App model_executor:
2022-09-20 11:39:59.136548 WARNING model_executor: Worker Ags: {'id': '85a2c90f1998400bb60d247b0fc9bf25', 'name': 'model_executor', 'objectid': '4c322dbd6a6043c1ba9f923dcd5bfa44', 'type': 'state', 'function': <bound method ModelExecutor.state_handler of <model_executor.ModelExecutor object at 0x7fa36aa8e0>>, 'attribute': 'state', 'entity': 'sensor.shellypro4pm_c8c9a3efe564_switch_1_energy', 'new_state': '9.06', 'old_state': '9.05', 'pin_app': True, 'pin_thread': 0, 'kwargs': {'__thread_id': 'thread-0'}}
2022-09-20 11:39:59.137617 WARNING model_executor: ------------------------------------------------------------
2022-09-20 11:39:59.139519 WARNING model_executor: Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/appdaemon/threading.py", line 917, in worker
    funcref(
  File "/thesillyhome_src/appdaemon/apps/model_executor.py", line 249, in state_handler
    feature_list = self.get_base_columns()
  File "/thesillyhome_src/appdaemon/apps/model_executor.py", line 220, in get_base_columns
    base_columns = pd.read_pickle(
  File "/usr/local/lib/python3.9/site-packages/pandas/io/pickle.py", line 187, in read_pickle
    with get_handle(
  File "/usr/local/lib/python3.9/site-packages/pandas/io/common.py", line 798, in get_handle
    handle = open(handle, ioargs.mode)
FileNotFoundError: [Errno 2] No such file or directory: '/thesillyhome_src/data/parsed/act_states.pkl'
2022-09-20 11:39:59.140654 WARNING model_executor: ------------------------------------------------------------
2022-09-20 11:39:59.151656 INFO model_executor: 
2022-09-20 11:39:59.154231 INFO model_executor: <--- sensor.shellypro4pm_c8c9a3efe564_switch_2_energy is 4.58 --->
2022-09-20 11:39:59.155271 WARNING model_executor: ------------------------------------------------------------
2022-09-20 11:39:59.156066 WARNING model_executor: Unexpected error in worker for App model_executor:
2022-09-20 11:39:59.156753 WARNING model_executor: Worker Ags: {'id': '85a2c90f1998400bb60d247b0fc9bf25', 'name': 'model_executor', 'objectid': '4c322dbd6a6043c1ba9f923dcd5bfa44', 'type': 'state', 'function': <bound method ModelExecutor.state_handler of <model_executor.ModelExecutor object at 0x7fa36aa8e0>>, 'attribute': 'state', 'entity': 'sensor.shellypro4pm_c8c9a3efe564_switch_2_energy', 'new_state': '4.58', 'old_state': '4.57', 'pin_app': True, 'pin_thread': 0, 'kwargs': {'__thread_id': 'thread-0'}}
2022-09-20 11:39:59.157296 WARNING model_executor: ------------------------------------------------------------
2022-09-20 11:39:59.158169 WARNING model_executor: Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/appdaemon/threading.py", line 917, in worker
    funcref(
  File "/thesillyhome_src/appdaemon/apps/model_executor.py", line 249, in state_handler
    feature_list = self.get_base_columns()
  File "/thesillyhome_src/appdaemon/apps/model_executor.py", line 220, in get_base_columns
    base_columns = pd.read_pickle(
  File "/usr/local/lib/python3.9/site-packages/pandas/io/pickle.py", line 187, in read_pickle
    with get_handle(
  File "/usr/local/lib/python3.9/site-packages/pandas/io/common.py", line 798, in get_handle
    handle = open(handle, ioargs.mode)
FileNotFoundError: [Errno 2] No such file or directory: '/thesillyhome_src/data/parsed/act_states.pkl'
2022-09-20 11:39:59.158967 WARNING model_executor: ------------------------------------------------------------
2022-09-20 11:40:02.392014 INFO model_executor: 
2022-09-20 11:40:02.393737 INFO model_executor: <--- sensor.shelly1_485519ca2e97_temperature_2 is 13.4 --->
2022-09-20 11:40:02.394429 WARNING model_executor: ------------------------------------------------------------
2022-09-20 11:40:02.395181 WARNING model_executor: Unexpected error in worker for App model_executor:
2022-09-20 11:40:02.396355 WARNING model_executor: Worker Ags: {'id': '85a2c90f1998400bb60d247b0fc9bf25', 'name': 'model_executor', 'objectid': '4c322dbd6a6043c1ba9f923dcd5bfa44', 'type': 'state', 'function': <bound method ModelExecutor.state_handler of <model_executor.ModelExecutor object at 0x7fa36aa8e0>>, 'attribute': 'state', 'entity': 'sensor.shelly1_485519ca2e97_temperature_2', 'new_state': '13.4', 'old_state': '13.2', 'pin_app': True, 'pin_thread': 0, 'kwargs': {'__thread_id': 'thread-0'}}
2022-09-20 11:40:02.397219 WARNING model_executor: ------------------------------------------------------------
2022-09-20 11:40:02.398497 WARNING model_executor: Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/appdaemon/threading.py", line 917, in worker
    funcref(
  File "/thesillyhome_src/appdaemon/apps/model_executor.py", line 249, in state_handler
    feature_list = self.get_base_columns()
  File "/thesillyhome_src/appdaemon/apps/model_executor.py", line 220, in get_base_columns
    base_columns = pd.read_pickle(
  File "/usr/local/lib/python3.9/site-packages/pandas/io/pickle.py", line 187, in read_pickle
    with get_handle(
  File "/usr/local/lib/python3.9/site-packages/pandas/io/common.py", line 798, in get_handle
    handle = open(handle, ioargs.mode)
FileNotFoundError: [Errno 2] No such file or directory: '/thesillyhome_src/data/parsed/act_states.pkl'
2022-09-20 11:40:02.399304 WARNING model_executor: ------------------------------------------------------------

**Comment by lcmchris** [comment]
Hi @dbabin ,
Thanks for raising this! The error here showing the missing act_states.pkl indicates that the parsing has failed.
To help me triage, could you let me know a few things:
1) Are you using this as an Addon or Container?
2) Are you using the latest dev version or a particular release version? (I did recently make a few fixes to Dev)
3) Could you share your /thesillyhome_src/data/config/options.json and /thesillyhome_src/log/thesillyhome.log with me?

It would be awesome if you could join the [support discord](https://discord.gg/YJxYCUGV) , as I can give much better help :)!

**Comment by dbabin** [comment]
1. Addon
2. Stable not dev; but tried dev as well - I think issue was there as well - I’ll try Dev again - I stopped the dev when I realized I needed to give write access to the database - had originally only given read accesss. I will change access rights to database and try dev agian. 

3. Will work to find where this is and send - I use a Raspberry pi hass and wasn’t sure where the silly home folder was. 

* will get on discord soon
